src/main/java/uk/gov/register/presentation/resource/DataResource.java

```diff
@@ -50,19 +51,21 @@ public Response downloadTorrent() {
     @GET
     @Path("/feed")
     @Produces({MediaType.TEXT_HTML, MediaType.APPLICATION_JSON, ExtraMediaType.TEXT_YAML, ExtraMediaType.TEXT_CSV, ExtraMediaType.TEXT_TSV, ExtraMediaType.TEXT_TTL})
-    public EntryListView feed() {
-        return viewFactory.getEntryFeedView(
-                queryDAO.getAllEntries(ENTRY_LIMIT)
-        );
+    public EntryListView feed(@QueryParam("pageIndex") long pageIndex, @QueryParam("pageSize") long pageSize) {
+        Pagination pagination = new Pagination(pageIndex, pageSize, queryDAO.getTotalEntriesCount());
+
+        List<DbEntry> entries = queryDAO.getAllEntries(pagination.pageSize(), pagination.offset());
```

Comment:
you could pass the whole `Pagination` object to the DAO and use the [`@BindBean` annotation](http://jdbi.org/sql_object_api_argument_binding/) to pull out the size and offset values. Do you think that might be a little cleaner?